libs/blocks/table/table.js

```diff
@@ -508,6 +516,7 @@ function applyStylesBasedOnScreenSize(table, originTable) {
       table.parentElement.insertBefore(filters, table);
       table.parentElement.classList.add(`table-${table.classList.contains('merch') ? 'merch-' : ''}section`);
       filterChangeEvent();
+      setAriaLabelsForElements(colSelect0, colSelect1);
```

Comment:
I'd like to future-proof this a bit, in case we'll have another number of columns visible on different breakpoints. So let's send these in an Array and have a single argument for `setAriaLabelsForElements` that iterates over the array's elements to set the `aria-label`.